### PR TITLE
Improve the plug-in cert task.

### PR DESCRIPTION
**Pull request by myidpt**
Improve the plugin CA certificate security task.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ X ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md [hunk 1]
```diff
@@ -30,12 +30,15 @@ to provision certificates and keys for any number of Istio CAs.
 
 ## Plug in certificates and key into the cluster
 
-{{< tip >}}
-For production cluster setup, it is a good practice to do the following on an offline machine with good
-security protection. The root private key should be exposed to as few people and processes as possible.
-{{< /tip >}}
+{{< warning >}}
+The following instructions are for demo purpose only.
```
Comment (smawson) [inline content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md:34]:
```suggestion
The following instructions are for demo purposes only.
```

File: content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md [hunk 2]
```diff
@@ -30,12 +30,15 @@ to provision certificates and keys for any number of Istio CAs.
...
+For production cluster setup, it is highly recommended to use a production-ready CA, such as
```
Comment (smawson) [inline content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md:35]:
```suggestion
For a production cluster setup, it is highly recommended to use a production-ready CA, such as
```

File: content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md [hunk 3]
```diff
@@ -30,12 +30,15 @@ to provision certificates and keys for any number of Istio CAs.
...
+[Hashicorp Vault](https://www.hashicorp.com/products/vault).
+It is a good practice to manage the root CA on an offline machine with good
```
Comment (smawson) [inline content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md:37]:
```suggestion
It is a good practice to manage the root CA on an offline machine with strong
```

File: content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md [hunk 4]
```diff
@@ -30,12 +30,15 @@ to provision certificates and keys for any number of Istio CAs.
...
+security protection.
+{{< /warning >}}
 
-1.  Create a directory for holding certificates and keys:
+1.  On the top-level directory of the Istio installation package, create a directory for holding certificates and keys:
```
Comment (smawson) [inline content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md:41]:
```suggestion
1.  In the top-level directory of the Istio installation package, create a directory to hold certificates and keys:
```

File: content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md [hunk 5]
```diff
@@ -69,12 +72,9 @@ security protection. The root private key should be exposed to as few people and
     * `root-cert.pem`: the root certificate
 
     {{< tip >}}
-    You can replace `cluster1` with a string of your choosing. For example, `make mycluster-certs` will
-    result in the creation of a directory called `mycluster`.
-    {{< /tip >}}
-
-    {{< tip >}}
-    To configure additional Istio CAs, you can repeat this step with different cluster/directory names.
+    To configure additional Istio CAs, you can repeat this step with different cluster names.
```
Comment (smawson) [inline content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md:75]: Should we say something about why they would do this, i.e. to setup additional CAs in other clusters? It is implied based on the variable names but we should be explicit here, and state that each cluster that has a control plane should have its own intermediate CA. We could say this at the top of the doc and then refer to it here.

**Review by myidpt — COMMENTED** [review]
Apply suggested modifications.

**Comment by smawson** [comment]
I only see some of the fixes, not all?

**Comment by myidpt** [comment]
@smawson Just pushed the new update. Sorry for the delay.

**Review by myidpt — COMMENTED** [review]
Apply recommended changes.

**Review by myidpt — COMMENTED** [review]
Apply recommended changes.

File: content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md [hunk 6]
```diff
@@ -12,30 +12,33 @@ test: yes
 This task shows how administrators can configure the Istio certificate authority (CA) with a root certificate,
 signing certificate and key.
 
-By default, Istio's CA generates a self-signed root certificate and key, and uses them to sign the workload certificates.
-Istio's CA can also sign workload certificates using an administrator-specified certificate and key, and with an
-administrator-specified root certificate.
+By default, the Istio CA generates a self-signed root certificate and key, and uses them to sign the workload certificates.
+If you want to better protect the root CA key, you should use a root CA which runs on a secure machine offline,
```
Comment (smawson) [inline content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md:16]:
```suggestion
To protect the root CA key, you should use a root CA which runs on a secure machine offline,
```

File: content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md [hunk 7]
```diff
@@ -12,30 +12,33 @@ test: yes
 This task shows how administrators can configure the Istio certificate authority (CA) with a root certificate,
 signing certificate and key.
 
-By default, Istio's CA generates a self-signed root certificate and key, and uses them to sign the workload certificates.
-Istio's CA can also sign workload certificates using an administrator-specified certificate and key, and with an
-administrator-specified root certificate.
+By default, the Istio CA generates a self-signed root certificate and key, and uses them to sign the workload certificates.
```
Comment (smawson) [inline content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md:15]:
```suggestion
By default the Istio CA generates a self-signed root certificate and key and uses them to sign the workload certificates.
```

**Review by smawson — COMMENTED** [review]
Thanks this looks much better, a couple little nits on the new text :)

**Comment by myidpt** [comment]
@smawson Thanks for the comments!

**Review by smawson — APPROVED** [review]

**Pull request merged**